docs/changelog.md

```diff
@@ -1,7 +1,10 @@
 0.15.1.4
+
 - Improved error reporting during venv metadata inspection.
 - [bugfix] Fixed incompatibility with pypy as venv interpreter (#372).
 - [bugfix] Replaced implicit dependency on setuptools with an explicit dependency on packaging (#339).
+- [refactor] Moved all commands to separate files within the commands module (#255).
+- [refactor] Command `ensurepath` renamed to `ensure_path`.
```

Comment:
Remove this so we don't confuse users (the subcommand is remaining 'ensurepath'.

Reply:
(Just remove the last bullet item referring to ensurepath.)

Reply:
Removed in c663d90